streaming/base/format/mds/encodings.py

```diff
@@ -466,6 +466,28 @@ def decode(self, data: bytes) -> Image.Image:
 class JPEG(Encoding):
     """Store PIL image as JPEG."""
 
+    def __init__(self, quality: int = 75):
+        assert 0 <= quality <= 100
+        self.quality = quality
+
+    @classmethod
+    def from_str(cls, text: str) -> Self:
+        """Parse this encoding from string.
+
+        Args:
+            text (str): The string to parse.
+
+        Returns:
+            Self: The initialized Encoding.
+        """
+        args = text.split(':') if text else []
+        assert len(args) in {0, 1}
+        if len(args) == 1:
+            quality = int(args[0])
+        else:
+            quality = 75
```

Comment:
is this a constant we can import from the PIL library? would like to adhere to whatever they do if possible

Reply:
what does "text" look like? if it is "JPEG:60", is quality not "quality = int(args[1])?"

Reply:
I think from_str only passes stuff after the colon.

Reply:
@cabreraalex mind clarifying @XiaohanZhangCMU 's comment?